plugins/invidious.py

```diff
@@ -54,10 +55,12 @@ def _get_items(self, entries: SelectorList) -> list[PodcastItem]:
 
     def _get_item(self, entry: Selector):
         video_id = entry.css('videoId::text').get()
+        description = entry.css('description::text').get()
         return PodcastItem(
             item_id=video_id,
             title=entry.css('title::text').get(),
-            description=entry.css('description::text').get(),
+            description=description and ftfy.fix_text(description),
```

Comment:
What invidious calls RSS feed (really an atom feed) seems a bit broken right now. It is reported as UTF-8. It must be interpreted as such, for title and other fields, or otherwise they would be printed as broken (it is a previously fixed bug when we switched from requests to httpx). However, I've seen that at least for Spanish channels specifically the description comes as latin-1, contrary to reported. The best approach for now is ftfy which applies an heuristic to fix encoding, so we don't break channels in other languages, until invidious improves their feed generation.